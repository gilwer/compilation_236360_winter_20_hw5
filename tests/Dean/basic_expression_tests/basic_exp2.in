void main(){
	printi(1+1);
	 
	printi(30-20);
	 
	printi(3*5);
	 
	printi(50/2);
	 
	printi(49/2);
}