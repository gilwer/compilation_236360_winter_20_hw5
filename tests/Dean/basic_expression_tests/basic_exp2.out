2
10
15
25
24
